### PR TITLE
Form-filler example

### PR DESCRIPTION
**Pull request by gabrielhuang**
This PR introduces the form filler agent. See the [readme](https://github.com/ServiceNow/TapeAgents/blob/formfiller/examples/form_filler/README.md) for more details

### CODE REVIEW

**Review by rizar — APPROVED** [review]
- please consider #137 and my comments here!
- looks very good!
- happy to merge after the tests pass, I can also help fixing the tests

File: examples/form_filler/README.md [hunk 1]
```diff
@@ -0,0 +1,222 @@
+# GREADTH Agent
+The GREADTH Agent is an agent that fills a form and is great at it.
+GREADTH stands for Grounded, REsponsive, Accurate, Disciplined, Transparent and Helpful.
+
+## Requirements
+- Make sure to activate the `tapeagents` conda environment.
+- Install additional requirements for this example `pip install -r requirements.formfiller.txt`
```
[the remark this answers is not in the document]
Comment (gabrielhuang) [reply examples/form_filler/README.md:7]: I don't see pyyaml or tqdm in any other requirements.txt file?

[the remark this answers is not in the document]
Comment (gabrielhuang) [reply examples/form_filler/README.md:7]: neither in pyproject.toml

[the remark this answers is not in the document]
Comment (rizar) [reply examples/form_filler/README.md:7]: yeah, but they are pulled somehow through other packages... they are already used in other parts of the framework

perhaps the right would be to add them as TapeAgents dependency, what's your take, @ollmer?

[the remark this answers is not in the document]
Comment (ollmer) [reply examples/form_filler/README.md:7]: I agree, let's add pyyaml and tqdm to the main requirements.txt. They're quite universal.

**Pull request merged**